README.md

```diff
@@ -1,518 +1,57 @@
-# Stripe Node.js Library
+# stripe-search-beta
 
-[![Version](https://img.shields.io/npm/v/stripe.svg)](https://www.npmjs.org/package/stripe)
-[![Build Status](https://travis-ci.org/stripe/stripe-node.svg?branch=master)](https://travis-ci.org/stripe/stripe-node)
-[![Coverage Status](https://coveralls.io/repos/github/stripe/stripe-node/badge.svg)](https://coveralls.io/github/stripe/stripe-node)
-[![Downloads](https://img.shields.io/npm/dm/stripe.svg)](https://www.npmjs.com/package/stripe)
-[![Try on RunKit](https://badge.runkitcdn.com/stripe.svg)](https://runkit.com/npm/stripe)
+This is a branch of [stripe-node](https://github.com/stripe/stripe-node) specialized for the use of the [Search API beta](https://stripe.com/docs/search-api).
 
-The Stripe Node library provides convenient access to the Stripe API from
-applications written in server-side JavaScript.
+It is experimental, and support for this package will end on 2022-01-01.
 
-For collecting customer and payment information in the browser, use [Stripe.js][stripe-js].
+It is branched off of the 8.174.0 release of `stripe-node`, and acts as a complete replacement for the library.
 
-## Documentation
+We publish a separate [npm package](https://www.npmjs.com/package/stripe-search-beta), `stripe-search-beta`, with the contents of this branch.
 
-See the [`stripe-node` API docs](https://stripe.com/docs/api?lang=node) for Node.js.
+To use it
 
-See [video demonstrations][youtube-playlist] covering how to use the library.
-
-## Requirements
-
-Node 8, 10 or higher.
-
-## Installation
-
-Install the package with:
-
-```sh
-npm install stripe --save
-# or
-yarn add stripe
-```
-
-## Usage
-
-The package needs to be configured with your account's secret key, which is
-available in the [Stripe Dashboard][api-keys]. Require it with the key's
-value:
-
-<!-- prettier-ignore -->
-```js
-const stripe = require('stripe')('sk_test_...');
-
-stripe.customers.create({
-  email: 'customer@example.com',
-})
-  .then(customer => console.log(customer.id))
-  .catch(error => console.error(error));
+```shell
+$ npm uninstall stripe
+$ npm install stripe-search-beta
 ```
 
-Or using ES modules and `async`/`await`:
+And then inside your application code
 
-```js
-import Stripe from 'stripe';
-const stripe = new Stripe('sk_test_...');
-
-(async () => {
-  const customer = await stripe.customers.create({
-    email: 'customer@example.com',
-  });
-
-  console.log(customer.id);
-})();
+```diff
+-const stripe = require('stripe')("<your-api-key>")
++const stripe = require('stripe-search-beta')("<your-api-key>")
 ```
```

Comment:
This looks great! Does anything need to change in their typescript files? I don't _think_ so but double checking.

Reply:
They'll need to change the package they import from, but that should be it.